inst/safetyGraphics_app/global.R

```diff
@@ -12,11 +12,12 @@ library(DT)
 library(haven)
 library(tidyr)
 
-# create vector of all possible charts
-all_charts <- c("edish","safetyhistogram")
-include_charts <- getShinyOption("safetygraphics_charts")
-if (!is.null(include_charts)){
-  all_charts <- all_charts[all_charts %in% include_charts]
+# subset chartsMetadata if user requests it
+if (!is.null(getShinyOption("safetygraphics_charts"))){
+  chartsMetadata_incl <- chartsMetadata[chartsMetadata$chart %in% getShinyOption("safetygraphics_charts"), ]
```

Comment:
Do we really need to clone/subset the whole metadata object here? I was thinking we'd really just update the all_charts vector ...